src/ast/mod.rs

```diff
@@ -1536,6 +1536,10 @@ pub enum Statement {
         replace_into: bool,
         /// Only for mysql
         priority: Option<MysqlInsertPriority>,
+        /// Only for mysql
+        as_table: Option<ObjectName>,
```

Comment:
I think you may need to add an annotation here so it visits the relation name

```suggestion
        #[cfg_attr(feature = "visitor", visit(with = "visit_relation"))]
        as_table: Option<ObjectName>,
```

Reply:
The annotation was added.